price-feeder/go.mod

```diff
@@ -41,7 +41,7 @@ require (
 	github.com/ChainSafe/go-schnorrkel v0.0.0-20200405005733-88cbf1b4c40d // indirect
 	github.com/Djarvur/go-err113 v0.0.0-20210108212216-aea10b59be24 // indirect
 	github.com/GaijinEntertainment/go-exhaustruct/v2 v2.3.0 // indirect
-	github.com/Gravity-Bridge/Gravity-Bridge/module v1.5.3 // indirect
+	github.com/Gravity-Bridge/Gravity-Bridge/module v1.7.2 // indirect
```

Comment:
Note: This change doesn't seem to be a purely linting. Was it related?

Reply:
yes.  WIthout it, we couldn't use the go workspaces. 

We still use our fork.

Reply:
FYI, this branch is where all the action is right now.  

WIll do more with price feed tomorrow.

Reply:
viz:
https://github.com/umee-network/umee/blob/1309241296d46bd91a19957868044245136bf8e7/price-feeder/go.mod#L305

actual used code is unchanged.

Reply:
As noted in comment above, please rollback. Also, we have other old PR for using go workspaces, and this is not relevant to this PR (linting update)

Reply:
Hmmmmmmmmm.....
Ok I will pull that commit.

Reply:
yes, and not mangle in this PR things not related to linting (eg go workspaces, and a wrong gravity bridge version)

Reply:
up, let's undo go.mod for gravity bridge.